_posts/2021-02-23-new-users.md

```diff
@@ -0,0 +1,40 @@
+---
+site: freiburg
+title: We want to welcome all new Galaxy users!
+tags:
+  - data
+  - tools
+  - training
+supporters:
+  - denbi
+  - elixir
+---
+
+For all new Galaxy users, we want to summarize briefly our service.
+
+__With Galaxy we offer you free, automatized analyses of (-omics) data for your lab!__
+
+We would like to invite you to use our free data analysis gateway Galaxy for your daily work. To use Galaxy, you do not need any (bio)informatics skill or to install software. All you need is a browser such as Chrome or Firefox.
+
+Our Galaxy platform provides >2500 data analysis [tools](https://galaxyproject.eu/tools) for e.g. __sequence analyses, genome annotation, epigenetics, metabolomics, proteomics, metagenomics, statistics and machine learning, and numerous visualizations__.
+The Galaxy Europe Team of the University of Freiburg as part of [de.NBI](https://www.denbi.de) and [ELIXIR](https://elixir-europe.org/communities/galaxy) runs the (BMBF funded) European Galaxy Server as one of the biggest public bioinformatics platforms with 10,000 CPU cores and already more than 25,000 users.
+
+__Our offer for your lab:__
+- __Free access to our data analysis platform Galaxy__ for the analyses of your high-throughput data: upload your raw files and get your results fast by our provided automatized workflows
+- __Free access to necessary hardware resources__ incl. 250 GB storage - independent of the compute power of your local device
+- __Free workflows__ for transparent, reproducible and standardized analysis of your data
```

Comment:
```suggestion
- __Free workflows__ for transparent, reproducible and standardized analysis of your data.
```

Reply:
do we have a link for this? this one maybe? https://usegalaxy.eu/workflows/list_published